_includes/install_instructions/python.html

```diff
@@ -14,7 +14,8 @@ <h3>Python</h3>
     well.  Installing all of its research packages individually can be
     a bit difficult, so we recommend
     <a href="https://www.anaconda.com/products/individual">Anaconda</a>,
-    an all-in-one installer.
+    an all-in-one installer. Note that the Anaconda install process can 
+    take a while, so make sure to do it in advance of the workshop.
```

Comment:
I think it's worth quantifying the length of time - I'd also consider five minutes to be a while
```suggestion
    take 20 minutes or more, so make sure to do it in advance of the workshop.
```